pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -129,6 +131,11 @@ func (c *constraint) computeSecurityContext(ctx context.Context, a admission.Att
 	// get all constraints that are usable by the user
 	klog.V(4).Infof("getting security context constraints for pod %s (generate: %s) in namespace %s with user info %v", pod.Name, pod.GenerateName, a.GetNamespace(), a.GetUserInfo())
 
+	if !c.sccSynced() {
```

Comment:
no, this will hang forever.  Try 

```
wait.pollimmediatefor10seconds(1*time.second, 10*time.second, func(){
check}
)
if err != nil{
return a forbidden error tell me it happened timedout
```

Reply:
Updated